handler-proxy/src/main/java/io/netty/handler/proxy/HttpProxyHandler.java

```diff
@@ -172,7 +172,7 @@ protected Object newInitialMessage(ChannelHandlerContext ctx) throws Exception {
     }
 
     @Override
-    protected boolean handleResponse(ChannelHandlerContext ctx, Object response) throws HttpProxyConnectException {
+    protected boolean handleResponse(ChannelHandlerContext ctx, Object response) throws Exception {
```

Comment:
This was flagged as a breakage so I fixed it. That said I think this is a false positive as `HttpProxyHandler` is final so there should be no problem here in reality.